requirements/developer.txt

```diff
@@ -9,5 +9,5 @@ nvgpu==0.8.0; sys_platform == 'win32'
 grpcio
 protobuf
 grpcio-tools
-transformers==4.6.0
+transformers==4.11.0
```

Comment:
Does this new version still support the old api (ie. outputs = self.model(input_batch))?

Reply:
right

Reply:
@HamidShojanazeri If this is a breaking change, please include some details in the Readme. It will be good to also check with HF team if there can be a workaround so that people don't need to change code for things to work